docs/zetacore/managment_for_unused_gas_fee.md

```diff
@@ -0,0 +1,69 @@
+# Management for Unused Gas Fee
+
+## Overview
+
+This document explains how gas fees are managed when processing outbound transactions on connected chains, particularly focusing on how differences between estimated and actual gas usage are handled.
+The Gas fee is always paid in the Gas ZRC20 token of the connected chain ,irrespective of coin type.
+
+## Gas Fee Scenarios
+
+When processing outbound transactions, there are two possible scenarios regarding gas fees:
+
+1. **User Overpays** (Common)
```

Comment:
- They overpay because the entire gas limit is consumed.
- They might underpay because the gas price might be increased by the protocol through the gas stability mechanism